src/js/containers/app/App.jsx

```diff
@@ -104,9 +116,14 @@ const mapDispatchToProps = (dispatch) => {
 const mapStateToProps = (state) => {
   return {
     movies: state.movies.data,
+    loadingStatus: state.movies.status,
     searchby: state.search.searchby,
     sortby: state.sortby,
     isLoading: state.movies.status === 'LOADING',
+    isLoadingSuccess: state.movies.status === 'LOAD_MOVIES_SUCCESS',
```

Comment:
could you plaese explain what do you think. it is a simple string comparision. i think the reselect is too complicated for that.

Reply:
Ok,
`loadingStatus` is not used anywhere
`isLoading` is not used anywhere

`state.movies.status === 'LOAD_MOVIES_SUCCESS'`
It's a simple string comprasion - I agree on that but this kind of stuff it's a performance killer, I suggested reselect because it provide a memoization pattern out of the box for this kind of situation.

So for re-render of component as selector would be called it just looks what's the value and returns memoized value of selector - without another comprasion at another re-render (somtimes you can see that re-render would be triggered for a many times especially with SPA)